XLS-0066d-lending-protocol/README.md

```diff
@@ -0,0 +1,1560 @@
+| `Owner`                |       `No`       |   `Yes`   | :heavy_check_mark: | `string`  |  `AccountID`  |     `N/A`     | The address of the Loan Broker account.                                                                                                                                                                      |
+| `Data`                 |      `Yes`       |   `No`    |                    | `string`  |    `BLOB`     |     None      | Arbitrary metadata about the `LoanBroker`. Limited to 256 bytes.                                                                                                                                             |
+| `ManagementFeeRate`    |       `No`       |   `Yes`   |                    | `number`  |   `UINT16`    |       0       | The 1/10th basis point fee charged by the Lending Protocol. Valid values are between 0 and 10000 inclusive. A value of 1 is equivalent to 1/10 bps or 0.001%                                                 |
+| `OwnerCount`           |       `No`       |   `No`    | :heavy_check_mark: | `number`  |   `UINT32`    |       0       | The number of active Loans issued by the `LoanBroker`.                                                                                                                                                       |
+| `DebtTotal`            |       `No`       |   `No`    | :heavy_check_mark: | `number`  |   `NUMBER`    |       0       | The total asset amount the protocol owes the Vault, including interest.                                                                                                                                      |
+| `DebtMaximum`          |      `Yes`       |   `No`    | :heavy_check_mark: | `number`  |   `NUMBER`    |       0       | The maximum amount the protocol can owe the Vault. The default value of 0 means there is no limit to the debt.                                                                                               |
+| `CoverAvailable`       |       `No`       |   `No`    | :heavy_check_mark: | `number`  |   `NUMBER`    |       0       | The total amount of first-loss capital deposited into the Lending Protocol.                                                                                                                                  |
+| `CoverRateMinimum`     |       `No`       |   `Yes`   | :heavy_check_mark: | `number`  |   `UINT16`    |       0       | The 1/10th basis point of the `DebtTotal` that the first loss capital must cover. Valid values are between 0 and 100000 inclusive. A value of 1 is equivalent to 1/10 bps or 0.001%.                         |
+| `CoverRateLiquidation` |       `No`       |   `Yes`   | :heavy_check_mark: | `number`  |   `UINT16`    |       0       | The 1/10th basis point of minimum required first loss capital that is liquidated to cover a Loan default. Valid values are between 0 and 100000 inclusive. A value of 1 is equivalent to 1/10 bps or 0.001%. |
```

Comment:
`CoverRateMinimum` and `CoverRateLiquidation` need to be `UINT32`, because max `UINT16` is 65535, and the max for these is 100,000. Also the case in the `LoanBrokerSet` transaction.

Reply:
Done!